src/components/views/beacon/RoomLiveShareWarning.tsx

```diff
@@ -57,6 +60,13 @@ const RoomLiveShareWarningInner: React.FC<RoomLiveShareWarningInnerProps> = ({ l
 
     const hasError = hasStopSharingError || hasLocationPublishError;
 
+    // eat events from buttons so navigate to tile
+    // is not triggered
+    const stopPropagationWrapper = (callback: () => void) => (e?: ButtonEvent) => {
+        e?.stopPropagation();
+        callback();
+    };
+
```

Comment:
this might be worth breaking out to a util somewhere, particularly with a `preventDefault` call, though this can be later work.